tools/doc/html.js

```diff
@@ -352,9 +352,13 @@ function linkJsTypeDocs(text) {
 }
 
 function parseAPIHeader(text) {
+  const classNames = 'api_stability api_stability_$2';
+  const docsUrl = 'documentation.html#documentation_stability_index';
+
   text = text.replace(
     STABILITY_TEXT_REG_EXP,
-    '<pre class="api_stability api_stability_$2">$1 $2$3</pre>'
+    `<pre class="${classNames}"><a href="${docsUrl}">$1 $2</a>$3
```

Comment:
I've updated to bring the closing `</pre>` to the same line, but I can't write it nested the way you have it because of the default formatting of a `<pre>` tag. Doing it the way you had it adds another 20px-ish left padding into the bar.